.github/workflows/pkgci_unit_test.yml

```diff
@@ -60,3 +60,22 @@ jobs:
           source ${VENV_DIR}/bin/activate
           echo "Testing compiler package:"
           python -m iree.compiler._package_test
+
+      # Test tools and importers that require additional deps.
+      - name: Test ONNX importer
+        run: |
+          source ${VENV_DIR}/bin/activate
+          python -m pip install onnx>=1.16.0
+          python compiler/bindings/python/test/tools/import_onnx_test.py
+          python compiler/bindings/python/test/extras/onnx_importer_test.py
```

Comment:
Tested with a known breaking change. The test fails as expected: https://github.com/iree-org/iree/actions/runs/10690956944/job/29636993342?pr=18420#step:9:19

However...

```
  File "/home/runner/work/iree/iree/compiler/bindings/python/test/extras/fx_importer_test.py", line 15, in <module>
    raise ModuleNotFoundError(
ModuleNotFoundError: Failed to import the fx_importer (for a reason other than torch not being found)
```

That's missing the actual error message :P. I had a local patch to the test that got more info. Need to dig that back up...

Reply:
There we go, got the right error message: https://github.com/iree-org/iree/actions/runs/10691656074/job/29638920091?pr=18420#step:9:19

```
Traceback (most recent call last):
  File "/home/runner/work/iree/iree/compiler/bindings/python/test/extras/fx_importer_test.py", line 8, in <module>
    from iree.compiler.extras import fx_importer
  File "/home/runner/work/iree/iree/.venv/lib/python3.11/site-packages/iree/compiler/extras/fx_importer.py", line 138, in <module>
    from .._mlir_libs._torchMlir import get_int64_max, get_int64_min
ModuleNotFoundError: No module named 'iree.compiler._mlir_libs._torchMlir'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/runner/work/iree/iree/compiler/bindings/python/test/extras/fx_importer_test.py", line [19](https://github.com/iree-org/iree/actions/runs/10691656074/job/29638920091?pr=18420#step:9:20), in <module>
    raise ModuleNotFoundError(
ModuleNotFoundError: Failed to import the fx_importer (for a reason other than torch not being found)
```